dotCMS/src/main/java/com/dotcms/contenttype/business/FieldFactoryImpl.java

```diff
@@ -110,6 +111,15 @@ public FieldVariable loadVariable(String id) throws DotDataException {
   @Override
   public FieldVariable save(FieldVariable fieldVar) throws DotDataException {
     return LocalTransaction.wrapReturn(() -> {
+      
+      if(!UtilMethods.isSet(fieldVar.key())){
```

Comment:
You can probably do the validation out of the local transaction, could be less expensive